Run dbt rpc when destination is not supported (#61)
fivetran/dbt-language-server: README.md

# dbt Wizard

This extension will help you work with [dbt](https://www.getdbt.com/).

> Also it provides additional features for BigQuery data warehouses like [Highlighting errors](#highlighting-errors), [Completion for SQL](#completion-for-sql) and etc.

Author: Plural seems better.

> Also it provides additional features for BigQuery data warehouses like [Highlighting errors](#highlighting-errors), [Completion for SQL](#completion-for-sql) and etc.

Reviewer: ```suggestion
Also it provides additional features like [Highlighting errors](#highlighting-errors), [Completion for SQL](#completion-for-sql), etc., for BigQuery data warehouses , 
```

# dbt Wizard

This extension will help you work with [dbt](https://www.getdbt.com/).

> Also it provides additional features like [Highlighting errors](#highlighting-errors), [Completion for SQL](#completion-for-sql), etc., for BigQuery data warehouses , 

Author: Fixed by @alex-ilyichov.

## How to use

The extension works on MacOS, Linux, and Windows (for Windows, see the additional steps in the [Windows Support](docs/WindowsSupport.md) article). It supports the default locations for `profiles.yml` and `dbt_project.yml` files.

> Prior to using the extension, you need to perform the following steps (**note**: replace *dbt-bigquery* with your adapter like *dbt-postgres*):

Reviewer: ```suggestion
Prior to using the extension, you need to perform the following steps (**note**: replace *dbt-bigquery* with your adapter, such as *dbt-postgres*):
```

> Prior to using the extension, you need to perform the following steps (**note**: replace <u><i>dbt-bigquery</i></u> with your adapter like <i><u>dbt-postgres</i></u>):

Author: Fixed. Now it is just italic.

> 1. [Install bigquery-dbt and dbt-rpc](https://docs.getdbt.com/dbt-cli/installation) (e.g `python3 -m pip install dbt-bigquery dbt-rpc`).

Reviewer: ```suggestion
1. [Install bigquery-dbt and dbt-rpc](https://docs.getdbt.com/dbt-cli/installation) (e.g., `python3 -m pip install dbt-bigquery dbt-rpc`).
```
